### DISCUSSION TITLE
Filter function and computed method help

### DISCUSSION THREAD

**Discussion by sfakhrym** [quasarframework/quasar#9191] [General - Components / Directives / etc] [answered]
Hello Quasar Friends :)

This question is a follow up from this discussion https://github.com/quasarframework/quasar/discussions/9155 where I got the clue for my problem.

But until now, I am still confuse because it doesn't show the result I expected (I already created the CodePen, please, help me, guys :) )

https://codepen.io/fakhruddinmasse/pen/GRWRQgX

1. The expected result is the list of items from the arrays inside variable `focuses1` and `focuses2` will be displayed in the review lists section, and sort it according to their selected priority (either 1st, 2nd, or 3rd priority).
2. I want to show only the title from the arrays

What I have done, are:

1. I created the variable `const focusesAll` to collect all the array together from `const focuses1` and `const focuses2` 
2. I created the computed method using filter functions for each priority lists

    `const first = computed(() => focusesAll.value.filter(v => v.priority === '1st'));`
    `const second = computed(() => focusesAll.value.filter(v => v.priority === '2nd'));`
    `const third = computed(() => focusesAll.value.filter(v => v.priority === '3rd'));`

3. In the review lists for first, second, and third priority; I created the `{{ first.title }} `/ `{{ second.title }}` / `{{ third.title }}` and with `v-modal="first"` / `v-modal="second"` / `v-modal="third"` respectively.
4. And in the end, the result doesn't show the title for each item that shows from the list items according to their selected priority

Well, I hope my explanation is clear, thank you in advanced guys, will really appreciate of your help :)

**Comment by metalsadman** [comment, marked as answer]
@fakhruddinmasse 
you don't need to ref a function, you use concat to merge two arrays then filter.
https://codepen.io/metalsadman/pen/poevjve?editors=1010

**Reply by sfakhrym** [reply]
Thank you so much, sir, finally :)

Oh yeah! I will learn more about it, thanks :)
